qa/workunits/rados/test.sh

```diff
@@ -1,14 +1,37 @@
 #!/usr/bin/env bash
 set -ex
 
-parallel=1
-[ "$1" = "--serial" ] && parallel=0
+# ./test.sh                       # Default: parallel mode, 30-min timeout
+# ./test.sh --serial              # Serial mode, 30-min timeout
+# ./test.sh --crimson             # Crimson mode, 30-min timeout
+# ./test.sh --timeout 3600        # Parallel mode, 60-min timeout
+# ./test.sh --serial --timeout 60 # Serial mode, 1-min timeout
+# ./test.sh --crimson --timeout 0 # Crimson mode, no timeout
 
-# let crimson run in serial mode
-crimson=0
-[ "$1" = "--crimson" ] && parallel=0 && crimson=1
+# First argument must be either --serial or --crimson or nothing
+if [ "$1" = "--serial" ]; then
+    parallel=0
+    shift # Remove the first argument from the list so timeout can be processed next
+elif [ "$1" = "--crimson" ]; then
+    parallel=0
+    crimson=1
+    shift
+fi
```

Comment:
@Matan-B 

I edited the script such that it can take more than 1 argument without any order, what do you think?

Reply:
reviewed! :)